packages/aws-cdk-lib/aws-codepipeline/lib/pipeline.ts

```diff
@@ -325,7 +332,9 @@ export class Pipeline extends PipelineBase {
       public readonly pipelineArn = pipelineArn;
     }
 
-    return new Import(scope, id);
+    return new Import(scope, id, {
+      environmentFromArn: pipelineArn,
+    });
```

Comment:
Can you please add unit tests to verify that the imported pipelines resolve account and region correctly?